app/controllers/boardgames_controller.rb

```diff
@@ -13,9 +13,11 @@ def new
   end
 
   def create
+    @user = User.find(params[:user_id])
     @boardgame = Boardgame.create(boardgame_params)
+    raise
     if @boardgame.save
-    redirect_to boardgame_path(@boardgame)
+      redirect_to boardgame_path(@boardgame)
     else
       render :new
     end
```

Comment:
I think we need to change the controller boardgame params and add :photo to it e.g.:
def article_params
  params.require(:article).permit(:title, :body, :photo)
end